BD-2763: Inbox Vision works for random users only (#6790)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/inbox_vision.md

## Testing your email in Inbox Vision

> Inbox Vision works with random users and doesn't persist any custom user or other preview user settings. This means that if you enter a custom user in the preview field and run Inbox Vision, what's shown in Inbox Vision might be different from the message preview elsewhere. 

Author: ```suggestion
Inbox Vision works when **Random User** is selected for **Preview as a User**  and does not persist any custom user or other preview user settings. This means that if you select **Custom User**  and run Inbox Vision, what's shown in Inbox Vision may differ from the message preview elsewhere. 
```
